src/modules/physics/physics.c

```diff
@@ -171,7 +172,7 @@ World* lovrWorldCreate(float xg, float yg, float zg, bool allowSleep, const char
     world->tags[i] = malloc(size);
     memcpy(world->tags[i], tags[i], size);
   }
-  memset(world->masks, 0xff, sizeof(world->masks));
+  memset(world->masks, 0xffff, sizeof(world->masks));
```

Comment:
The clear value for `memset` is a single byte, so this does not need to be changed from `0xff`.